### ISSUE TITLE
[bug]: google auth callback not working on self host

### ISSUE THREAD

**Issue by Chronos77** [hoppscotch/hoppscotch#3989]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

when I launch dockers and I try to connect to the admin part with google, it redirects me to the callback inquire however that does nothing and it leaves me on the login page, without any error in the admin logs or backend.

### Steps to reproduce

1. start docker containers
2. update env variable 
3. try to connect admin console

### Environment

Deploy preview

### Version

Self-hosted

**Comment by balub** [comment]
Hey @Chronos77, please feel free to correct me if I got this wrong. After going through the complete login flow with Google successfully you are left back on the login screen not logged into Hoppscotch ?

If this is the case is your instance running on HTTP ? Hoppscotch uses HTTP secure cookies which means unless its HTTPS the cookies don't get saved in the browser.

**Comment by Chronos77** [comment]
Hello, thank you for your answer. 

This is indeed the case regarding my problem.

Currently on the instance, I am using ports 3100, 3170 and 3000. Then we set up a load balancer which redirects an HTTPS url to one of its ports.

I therefore suppose that we would have to run the hoppscotch server in https, is that right? Is there an easy way to put it in https?

**Comment by balub** [comment]
@Chronos77 yes, can you look into TLS certificates and how you can get them for your domain ?

**Comment by balub** [comment]
@Chronos77 are you still facing this issue?

**Comment by liyasthomas** [comment]
There have been a lot of changes to the platform. If you're still having problems with the latest version, please let us know.
